src/ontology/mondo-edit.obo

```diff
@@ -329202,18 +329192,15 @@ is_a: MONDO:0018158 {source="Orphanet:254803"} ! mitochondrial DNA depletion syn
 
 [Term]
 id: MONDO:0016797
-name: multiple mitochondrial DNA deletion syndrome
-def: "A mitochondrial oxidative phosphorylation disorder in which mitochondrial DNA (mtDNA) level is lower than normal." [PMID:24266892]
-comment: Reason of obsoletion: out of scope - MONDO:excludeGroupingClass. This is part of a large mass-obsoletion effort for ORDO terms.
-subset: disease_grouping
-subset: obsoletion_candidate
+name: obsolete multiple mitochondrial DNA deletion syndrome
+def: "OBSOLETE. A mitochondrial oxidative phosphorylation disorder in which mitochondrial DNA (mtDNA) level is lower than normal." [PMID:24266892]
 subset: ordo_group_of_disorders {source="Orphanet:254807"}
```

Comment:
should this `subset` be maintained?

Reply:
yes, I'll remove it, good catch, thanks!